fe/src/main/java/org/apache/doris/journal/JournalEntity.java

```diff
@@ -109,63 +109,73 @@ public void write(DataOutput out) throws IOException {
         data.write(out);
     }
 
-    @Override
     public void readFields(DataInput in) throws IOException {
-        boolean needRead = true;
```

Comment:
I think it's better to leave this value and add precondition check to make it true after switch clause. If some logic is forgetted to be changed, this can help us to avoid disaster.

Reply:
OK